cartoframes/viz/widget_list.py

```diff
@@ -46,10 +46,16 @@ def _init_widgets(self, widgets):
     def get_variables(self):
         variables = {}
         for widget in self.widgets:
-            if widget and widget.has_variable():
+            if widget and widget._value:
                 variables[widget._name] = widget._value
         return variables
 
+    def get_properties(self):
+        properties = {}
+        for widget in self.widgets:
+            properties[widget._prop] = '@' + widget._name
```

Comment:
So, if you define a filter here, outside of the bridge, you will lose it as soon as any other widget filters through the bridge.

Reply:
I'm trying to test it out locally, will post results

Reply:
Also, widgets don't have to define a prop (like the category or histogram widget), so this will not work for all

Reply:
Fixed